package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "cross-echo",
+	"version": "0.0.1",
+	"description": "Cross platform echo command",
+	"main": "index.js",
+	"scripts": {},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/cpuabuse/cross-echo.git"
+	},
+	"keywords": [
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected object keys to be in natural insensitive ascending order. 'keywords' should be before 'repository'.](https://app.codacy.com/gh/cpuabuse/cross-echo/pullRequest?prid=5213902)